tests/config.py

```diff
@@ -1,4 +1,5 @@
 [
+    Test(name="OscarForHomalg", script = "test-OscarForHomalg.sh"),
```

Comment:
Can we make the filename all lowercase for consistency?

(To be fair, I'm not sure if "all lowercase" is the best option here and whether we want to revisit this style choice, it's just what I used out of habit for Unix systems, but right now that's where we are.)

Reply:
I was aware of the inconsistency when I choose that name. The alternative `test-oscarforhomalg.sh` would be harder to read. My aim was to preserve the spelling of the original package name. For the same reason I choose `test-HomalgProject_jl.sh` in the past.

We already have some inconsistency, which is my fault:

Some tests for Julia packages end in "jl":

`test-gapjl.sh`
`test-singularjl.sh`
`test-polymakejl.sh`

one in "_jl"

`test-HomalgProject_jl.sh`

and some do not have a "jl" at all (probably because there is no version "without Julia")

`test-nemo.sh`
`test-absalg.sh`
 
BTW: Personally I do not like the abbreviation in `test-absalg.sh` very much.

My preference would be to choose names as close to the original package names as possible (with minimal changes such as replacing dots by underlines). I would also include the "jl" for Julia packages to indicate that those are Julia packages. The repository `GAP.jl` is special in the sense that is contains both a Julia package and GAP packages. I guess the GAP packages from `GAP.jl` are no tested until now, right? This partially happens with this PR. Where should further tests from the GAP side go? Also in `test-gapjl.sh`?

In the end I am flexible. If you insist on lowercase names, I will change it.

Reply:
Yeah, that's actually a good point. I think we should fix file names readability in the long term.

As for adding further tests on the GAP side, they should be split up in a way that makes sense.